apps/docs-app/docs/features/server/server-side-rendering.md

```diff
@@ -64,4 +64,30 @@ export default defineConfig(({ mode }) => ({
 }));
 ```
 
-You can opt out of prerendering by passing an empty array of routes.
+You can opt out of prerendering by passing an empty array of routes and disabling prerender on the root route.
+
+```js
+import { defineConfig } from 'vite';
+import analog from '@analogjs/platform';
+// https://vitejs.dev/config/
+export default defineConfig(({ mode }) => ({
+  // ...other config
+  plugins: [
+    analog({
+      ssr: true,
+      nitro: {
+        routeRules: {
+          '/': {
+            prerender: false,
+          },
+        },
+      },
+      prerender: {
+        routes: async () => {
+          return ['/'];
```

Comment:
```suggestion
          return [];
```

Reply:
Done.  FYI: without ['/'] the root url is still not server side rendered. I just tried it out to make sure.